packages/evals/src/cli/runTask.ts

```diff
@@ -156,6 +157,29 @@ export const runTask = async ({ run, task, publish, logger }: RunTaskOptions) =>
 	const cancelSignal = controller.signal
 	const containerized = isDockerContainer()
 
+	// Perform git pull to ensure we have the latest version of the repo
+	const repoConfig = getRepoConfig(workspacePath)
+	if (repoConfig) {
+		logger.info(`performing git pull for repo: ${repoConfig.name} at ${repoConfig.path}`)
+		try {
+			const gitPullResult = await execa("git", ["pull"], { cwd: repoConfig.path })
+			logger.info(`git pull completed successfully: ${gitPullResult.stdout}`)
+		} catch (error) {
+			logger.warn(`git pull failed for ${repoConfig.name}: ${error}`)
+			// Continue execution even if git pull fails
+		}
+	} else {
+		// Fallback: try git pull in the workspace directory
+		logger.info(`performing git pull in workspace: ${workspacePath}`)
+		try {
+			const gitPullResult = await execa("git", ["pull"], { cwd: workspacePath })
+			logger.info(`git pull completed successfully: ${gitPullResult.stdout}`)
+		} catch (error) {
+			logger.warn(`git pull failed in workspace ${workspacePath}: ${error}`)
+			// Continue execution even if git pull fails
+		}
+	}
```

Comment:
In if the `workspacePath` doesn't exactly match a repo path then we should run `git pull` on all repos.